### DISCUSSION TITLE
Self-hosted: github.com rate limit guidelines

### DISCUSSION THREAD

**Discussion by pyunramura** [renovatebot/renovate#16658] [Request Help] [answered]
I'm exploring running the self-hosted version of the renovate tool and have a small question regarding how often it's advisable to run it. I'm aware that while checking for updates, the tool hits github for changelogs, but I'm unsure how much traffic is being generated each time it runs. For reference, I have 4 repos and around 50 dependencies total that will be extracted each run. After setting a github token surely I won't be approching any rate liming, correct? If so, how short of a time period is safe to rerun the tool? Every 1 minute, 5 minutes, 30 minutes, etc.? Any input or a link to some relevant documentation would be appreciated.

**Comment by rarkins** [comment, marked as answer]
We recommend running hourly per repo

**Reply by pyunramura** [reply]
Thanks a lot!
